frontend/test/resolution/testModuleInitOrder.cpp

```diff
@@ -40,9 +40,151 @@ static std::string chplHome() {
   return ret;
 }
 
+static void
+checkMentionedModules(Context* ctx, ID idMod, ...) {
+  auto& v = findMentionedModules(ctx, idMod);
+
+  for (const auto& id : v) {
+    std::cout << "  " << id.str() << "\n";
+  }
+
+  va_list args;
+  va_start(args, idMod);
+
+  for (const auto& id : v) {
+    assert(!id.isEmpty());
+
+    auto ast = parsing::idToAst(ctx, id);
+    auto mod = ast ? ast->toModule() : nullptr;
+    assert(ast && mod);
```

Comment:
```suggestion
    auto ast = parsing::idToAst(ctx, id);
    assert(ast);
    auto mod = ast->toModule();
    assert(mod);
```

I've found that separating assertions helps see at a glance what went wrong without spinning up a debugger, and in this case, it also helps avoid the odd ternary.